frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-root-page/feature-flag-root-page-content/feature-flag-root-section-card/feature-flag-root-section-card.component.ts

```diff
@@ -88,6 +90,14 @@ export class FeatureFlagRootSectionCardComponent {
     );
   }
 
+  async getAllFeatureFlagIds() {
```

Comment:
Shouldn't need to involve promises, but the actual need for getting all entity ids is already solved by ngrx. see the feature flag reducer, we actually have `selectIds` available already automatically from ngrx feature store, so you can create a selector for this:

```
export const selectFeatureFlagIds = createSelector(
  selectFeatureFlagState,
  (state) => state.selectIds
);
```